include/wx/aui/auibar.h

```diff
@@ -150,87 +168,87 @@ class WXDLLIMPEXP_AUI wxAuiToolBarItem
+    virtual void SetShortHelp(const wxString& s) { m_shortHelp = s; }
+    virtual const wxString& GetShortHelp() const { return m_shortHelp; }
 
-    void SetLongHelp(const wxString& s) { m_longHelp = s; }
-    const wxString& GetLongHelp() const { return m_longHelp; }
+    virtual void SetLongHelp(const wxString& s) { m_longHelp = s; }
+    virtual const wxString& GetLongHelp() const { return m_longHelp; }
 
-    void SetMinSize(const wxSize& s) { m_minSize = s; }
-    const wxSize& GetMinSize() const { return m_minSize; }
+    virtual void SetMinSize(const wxSize& s) { m_minSize = s; }
+    virtual const wxSize& GetMinSize() const { return m_minSize; }
 
-    void SetSpacerPixels(int s) { m_spacerPixels = s; }
-    int GetSpacerPixels() const { return m_spacerPixels; }
+    virtual void SetSpacerPixels(int s) { m_spacerPixels = s; }
+    virtual int GetSpacerPixels() const { return m_spacerPixels; }
 
-    void SetProportion(int p) { m_proportion = p; }
-    int GetProportion() const { return m_proportion; }
+    virtual void SetProportion(int p) { m_proportion = p; }
+    virtual int GetProportion() const { return m_proportion; }
 
-    void SetActive(bool b) { m_active = b; }
-    bool IsActive() const { return m_active; }
+    virtual void SetActive(bool b) { m_active = b; }
+    virtual bool IsActive() const { return m_active; }
 
-    void SetHasDropDown(bool b)
+    virtual void SetHasDropDown(bool b)
     {
-        wxCHECK_RET( !b || m_kind == wxITEM_NORMAL,
+        wxCHECK_RET( !b || m_kind == wxAUI_TB_ITEM_NORMAL,
                      wxS("Only normal tools can have drop downs") );
 
         m_dropDown = b;
     }
 
-    bool HasDropDown() const { return m_dropDown; }
+    virtual bool HasDropDown() const { return m_dropDown; }
 
-    void SetSticky(bool b) { m_sticky = b; }
-    bool IsSticky() const { return m_sticky; }
+    virtual void SetSticky(bool b) { m_sticky = b; }
+    virtual bool IsSticky() const { return m_sticky; }
 
-    void SetUserData(long l) { m_userData = l; }
-    long GetUserData() const { return m_userData; }
+    virtual void SetUserData(long l) { m_userData = l; }
+    virtual long GetUserData() const { return m_userData; }
 
-    void SetClientData(wxObject* l) { m_clientData = l; }
-    wxObject* GetClientData() const { return m_clientData; }
+    virtual void SetClientData(wxObject* l) { m_clientData = l; }
+    virtual wxObject* GetClientData() const { return m_clientData; }
 
-    void SetAlignment(int l) { m_alignment = l; }
-    int GetAlignment() const { return m_alignment; }
+    virtual void SetAlignment(int l) { m_alignment = l; }
+    virtual int GetAlignment() const { return m_alignment; }
 
-    bool CanBeToggled() const
+    virtual bool CanBeToggled() const
     {
-        return m_kind == wxITEM_CHECK || m_kind == wxITEM_RADIO;
+        return m_kind == wxAUI_TB_ITEM_CHECK || m_kind == wxAUI_TB_ITEM_RADIO;
     }
 
-private:
+protected:
```

Comment:
Do we really need to do this? We already have accessors to all these fields AFAICS, so this doesn't seem necessary.